cdk/src/open-data-platform/data-plane/schema/schema.sql

```diff
@@ -30,131 +30,233 @@ $$ LANGUAGE plpgsql;
     private_lead_prediction float,
     -- Can be used to determine if the row needs updating with a new dataset.
-    last_updated timestamp default current_timestamp,
-    geom geometry(Geometry, 4326),
+    last_updated            timestamp default current_timestamp,
+    geom                    geometry(Geometry, 4326),
     PRIMARY KEY (address)
 );
 
 -- Either of these might be used for searching.
 CREATE INDEX IF NOT EXISTS geom_index ON parcels USING GIST (geom);
 
-CREATE TRIGGER update_last_update_timestamp BEFORE UPDATE ON parcels
-    FOR EACH ROW EXECUTE PROCEDURE update_last_update_timestamp();
+CREATE TRIGGER update_last_update_timestamp
+    BEFORE UPDATE
+    ON parcels
+    FOR EACH ROW
+EXECUTE PROCEDURE update_last_update_timestamp();
 
 -- States
-CREATE TABLE IF NOT EXISTS states(
+CREATE TABLE IF NOT EXISTS states
+(
     census_geo_id varchar(255) NOT NULL,
-    fips varchar(255) UNIQUE,
-    ansi varchar(255),
-    aff_geo_id varchar(255),
-    usps varchar(255),
-    name varchar(255),
-    lsad varchar(255),
-    geom GEOMETRY(Geometry, 4326),
-    PRIMARY KEY(census_geo_id)
-    );
+    fips          varchar(255) UNIQUE,
+    ansi          varchar(255),
+    aff_geo_id    varchar(255),
+    usps          varchar(255),
+    name          varchar(255),
+    lsad          varchar(255),
+    geom          GEOMETRY(Geometry, 4326),
+    PRIMARY KEY (census_geo_id)
+);
 
 CREATE INDEX IF NOT EXISTS geom_index ON states USING GIST (geom);
 
 -- Counties
-CREATE TABLE IF NOT EXISTS counties (
+CREATE TABLE IF NOT EXISTS counties
+(
     census_geo_id varchar(255) NOT NULL,
-    fips varchar(255) NOT NULL,
-    state_fips varchar(255) NOT NULL references states(fips),
-    ansi varchar(255) NOT NULL,
-    aff_geo_id varchar(255),
-    name varchar(255),
-    lsad varchar(255),
-    geom geometry(Geometry, 4326),
+    fips          varchar(255) NOT NULL,
+    state_fips    varchar(255) NOT NULL references states (fips),
+    ansi          varchar(255) NOT NULL,
+    aff_geo_id    varchar(255),
+    name          varchar(255),
+    lsad          varchar(255),
+    geom          geometry(Geometry, 4326),
     PRIMARY KEY (census_geo_id)
 );
-
 CREATE INDEX IF NOT EXISTS geom_index ON counties USING GIST (geom);
 
 -- Zipcodes
-CREATE TABLE IF NOT EXISTS zipcodes (
+CREATE TABLE IF NOT EXISTS zipcodes
+(
     census_geo_id varchar(255) NOT NULL,
-    zipcode varchar(255) NOT NULL,
-    lsad varchar(255) NOT NULL,
-    aff_geo_id varchar(255),
-    geom geometry(Geometry, 4326),
+    zipcode       varchar(255) NOT NULL,
+    lsad          varchar(255) NOT NULL,
+    aff_geo_id    varchar(255),
+    geom          geometry(Geometry, 4326),
     PRIMARY KEY (census_geo_id)
 );
 
 CREATE INDEX IF NOT EXISTS geom_index ON counties USING GIST (geom);
 CREATE UNIQUE INDEX IF NOT EXISTS zipcode_index ON zipcodes (zipcode);
 
+--- Returns lead connections aggregated by either state or water system, depending on z (zoom level).
```

Comment:
Did you say at standup you plan to do this on the client instead of in a function? If so, is this obsolete?

Reply:
I was going to add parcel data source + toggling in the FE (as opposed to adding it into this function source as a third option in the if/else). This is due to latency dragging a bit when I added a third source to this function. I was planning on keeping toggling between state + water system data in this function source.